### PR TITLE
Also display the total number of points for a given user.

### PR DESCRIPTION
**Pull request by StaticallyTypedRice**

### CODE REVIEW

File: ui/src/translations/de.ts [hunk 1]
```diff
@@ -74,6 +74,7 @@ export const de = {
     number_of_users: '{{count}} Benutzer',
     number_of_subscribers: '{{count}} Abonnenten',
     number_of_points: '{{count}} Punkte',
+    total_number_of_points: '{{count}} Gesamtpunkte',
```
Comment (dessalines) [inline ui/src/translations/de.ts]: Its not necessary to add another field here, number of points is already good.

**Comment by dessalines** [comment]
Thanks, I'll get this merged soon.

File: ui/src/translations/de.ts (hunk 1 above)
Comment (StaticallyTypedRice) [reply ui/src/translations/de.ts]: I wanted to make it clear that this metric was the total point count and not points for another type of content.

Do I still need to fix this not that you've merged it?

Comment (dessalines) [reply ui/src/translations/de.ts]: No I got it, I changed some other things around too.

**Pull request merged**